smithy-model/src/main/java/software/amazon/smithy/model/shapes/SmithyIdlModelSerializer.java

```diff
@@ -511,6 +541,18 @@ private boolean isEmptyStructure(Node node, Shape shape) {
             return !shape.isDocumentShape() && node.asObjectNode().map(ObjectNode::isEmpty).orElse(false);
         }
 
+        @Override
+        public Void enumShape(EnumShape shape) {
+            shapeWithMembers(shape, new ArrayList<>(shape.members()), true);
```

Comment:
Why the need to create a separate ArrayList of the members?

Reply:
Ah it was just because the method expects a list. There's no real need for it to be a list, just for it to be ordered. But since those are ordered anyway I changed it to collection.